.github/workflows/deploy.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node}}
+      - name: Cache Node.js modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.yarn
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
+      # - name: Publish Next
```

Comment:
Todo I wasn't sure what you wanted to do here. Maybe build and push that demo? I've noticed that it's also possible to build this as a client-only app right?

Reply:
This publishes the npm package as a 'next' version when run on the dev branch :)